docs/code/reference.md

```diff
@@ -140,6 +140,17 @@ than good.
 Also worth referring that there should be no Hungarian notation, and no mix of
 any kinds of notations.
 
+## Casting {#casting}
+
+Prevent using `as` keyword when casting as it might give `null` as result, which could be easily forgotten to check.
```

Comment:
From an language point of view, it might be better off said "Avoid using..." instead.